spec/features/variable_drafts/preview/invalid/acquisition_source_name_preview_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe 'Invalid Variable Draft Acquisition Source Name Preview', :js => true do
+  before do
+    login
+    @draft = create(:invalid_variable_draft, user: User.where(urs_uid: 'testuser').first)
```

Comment:
for the tests in this folder, I thought it was trying to test if there was invalid data in the draft. Are you able to add an invalid value in the draft factory being used here?